src/modules/projectContext.js

```diff
@@ -68,6 +69,26 @@ class ProjectContext {
      * @returns A new ProjectSearchContext
      */
     search = () => new ProjectSearchContext(this.sessionManager, this.id);
+
+    /**
+     * Get the data from a trace as array buffer.
+     *
+     * @param {string} traceUid The traceUid of the trace, format 'imageId:traceId', e.g. '093da8cb-77f8-46df-ac99-ea93aeede0be:0-1-1-a3f'
+     * @param {string} dataType The name of the data stream, as described in the trace, e.g. 'raw', 'text', 'ocr'
+     * @param {number} start Optional: The start of a subrange, inclusive. See spec https://tools.ietf.org/html/rfc7233#section-2.1
+     * @param {number} end Optional: The end of a subrange, inclusive. See spec https://tools.ietf.org/html/rfc7233#section-2.1
+     */
+     data = (traceUid, dataType, start = 0, end) => {
```

Comment:
Using an options object for the optionals feels more javascripty:

```suggestion
     data = (traceUid, dataType, {start = 0, end}) => {
```

Reply:
discussed